cocos/game/game.ts

```diff
@@ -880,10 +880,25 @@ export class Game extends EventTarget {
                     console.time('Init Project');
                 }
                 const jsList = querySettings<string[]>(SettingsCategory.PLUGINS, 'jsList');
+                if (!jsList) return Promise.resolve();
+
+                const getRootPath = (): string => {
+                    if (PREVIEW) {
+                        return 'plugins';
+                    }
+                    if (EDITOR && !EDITOR_NOT_IN_PREVIEW) {
+                        const server: string | null = querySettings<string>(SettingsCategory.ASSETS, 'server');
+                        if (server) {
+                            return `${server}plugins`;
```

Comment:
Could we make sure the `server` ends with `/` ?

Reply:
Yes, because this is what the editor does to initialize the configuration settings